slave/buildslave/pbutil.py

```diff
@@ -24,6 +24,8 @@
 from twisted.python import log
 from twisted.cred import error
 
+import klog
```

Comment:
Minor: It would be nice to not introduce new pep issues. My recommendation is to add an extra blank line between import statements and class declaration

Reply:
Done